diff -aurN e4rat-0.2.3.orig/e4rat.conf.sample e4rat-0.2.3/e4rat.conf.sample
--- e4rat-0.2.3.orig/e4rat.conf.sample	2012-11-08 23:45:23.552558829 +0100
+++ e4rat-0.2.3/e4rat.conf.sample	2012-11-09 00:15:52.588046169 +0100
@@ -14,7 +14,7 @@
 ; verbose  7
 
 ; Specify bit mask of log events written to disk (DEFAULT: 3)
-; loglevel  3
+loglevel  1
 
 ; collect only files which device type is ext4 (DEFAULT: true)
 ; ext4_only  true
@@ -26,7 +26,7 @@
 ; timeout  120
 
 ; path to init process binary (DEFAULT: /sbin/init)
-; init /sbin/init
+init /usr/bin/systemd
 
 ; Set target path log events should written to.
 ; If log_target is set so 'syslog' messages are send to syslogd.
